src/i18n/locales/en-US.json

```
@@ -240,6 +240,14 @@
 
     "trackSelection": {
       "title": "Select track(s) for processing",
+      "trackInactive": "inactive",
+      "deleteTrackText": "Delete Track",
+      "restoreTrackText": "Restore Track",
+      "cannotDeleteTrackText": "Cannot Delete Track",
+      "deleteTrackTooltip": "Do not encode and publish this track.",
+      "restoreTrackTooltip": "Encode and publish this track.",
+      "cannotDeleteTrackTooltip": "Cannot remove this track from publication.",
+      "noSelectionPossible": "Track selection is disabled for this video, because there are not enough tracks to deselect any.",
```

I would suggest providing a more precise description about the number of tracks required:

```suggestion
      "noSelectionPossible": "Track selection is disabled for this video, because at least two tracks are required to deselect any.",
```